plugins/BEdita/Core/tests/TestCase/Model/Table/ProfilesTableTest.php

```diff
@@ -129,6 +129,7 @@ public function validationProvider()
     public function testValidation($expected, array $data)
     {
         $profile = $this->Profiles->newEntity($data);
+        // TODO: should work commenting these 2 lines, but it doesn't... why???
```

Comment:
The problem appears to be in CTI's `ExtensionOf` association. It seems to me that there's a strange bug that causes two very different phenomena: both this and #1034. Hence, this should be a problem that's unrelated to this PR.